components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3652,7 +3736,25 @@ where
         }
     }
 
+    /// Propose batched raft commands(if any) first, then propose the given raft command.
     fn propose_raft_command(
```

Comment:
Seems the method is unnecessary at all. The only place it's used is https://github.com/tikv/tikv/pull/11147/files#diff-7a4718b9cddd27a4ef7e563024d5283cf14b4ba3abf26edc33b974b87680a5fbL587, which can be prepend batch propose call.

Reply:
Another place is `RollbackMerge`. This method is intended to be called when the order between this command and batched normal commands is necessary.